src/ocrd_models/ocrd_mets.py

```diff
@@ -883,8 +885,11 @@ def physical_pages_labels(self):
         return {div.get('ID'): (div.get('ORDER', None), div.get('ORDERLABEL', None), div.get('LABEL', None))
                 for div in divs}
 
-    def merge(self, other_mets, force=False, fileGrp_mapping=None, fileId_mapping=None, pageId_mapping=None,
-              after_add_cb=None, **kwargs):
+    def merge(self, other_mets, force : bool = False, 
```

Comment:
```suggestion
    def merge(self, other_mets : 'OcrdMets', force : bool = False, 
```

You can reference classes not yet defined by quoting them.